scripts/retrieve_databundle_light.py

```diff
@@ -354,6 +356,73 @@ def download_and_unzip_direct(config, rootpath, hot_run=True, disable_progress=F
     return True
 
 
+def download_and_unzip_hydrobasins(
+    config, rootpath, hot_run=True, disable_progress=False
+):
+    """
+    download_and_unzip_basins(config, rootpath, dest_path, hot_run=True,
+    disable_progress=False)
+
+    Function to download and unzip the data for hydrobasins.
```

Comment:
May be nice to add the path to the data and the licensing

Reply:
Well, it has been definitely a good idea to look into the license conditions:
> The following copyright statement must be displayed with, attached to or embodied in (in a reasonably prominent manner) the documentation or metadata of any Licensee Product or Program provided to an End User when utilizing the Licensed Materials: ...

Have added it to the docstring. Do you agree?